replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorRunner.java

```diff
@@ -52,23 +49,22 @@
   private static final String DEFAULT_REPLICATION_PERIOD_STR =
       String.valueOf(TimeUnit.MINUTES.toSeconds(5));
 
-  public ReplicatorRunner(
-      ScheduledExecutorService scheduledExecutor,
-      Replicator replicator,
-      ReplicatorConfigManager replicatorConfigManager) {
-    this(scheduledExecutor, replicator, replicatorConfigManager, Security.getInstance());
+  public ReplicatorRunner(Replicator replicator, ReplicatorConfigManager replicatorConfigManager) {
+    this(
+        Executors.newSingleThreadScheduledExecutor(
+            StandardThreadFactoryBuilder.newThreadFactory("replicatorRunnerThread")),
```

Comment:
For our standalone app I don't think we need to use the StandardThreadFactoryBuilder

Reply:
Can we not pass in an executor like before?

Reply:
Removed the threadFactoryBuilder and the platform util dependency. We now have no DDF dependencies in replicator-api-impl 🎉